### ISSUE TITLE
Abnormal CD8+ T cell subset proportion

### ISSUE THREAD

**Issue by adamklocperk** [obophenotype/human-phenotype-ontology#11327]
**Preferred term label:**
Abnormal CD8+ T cell subset proportion

**Synonyms**
same as the synonyms of Abnormal CD4+ T cell subset proportion HP:0031392, but with CD4 changed to CD8

**Definition (free text, please give PubMed ID)**
see above

**Parent term (use hpo.jax.org/app)**
Abnormal T cell subset distribution HP:0025540

**Child terms*
The following terms should be this term's children, INSTEAD of being the children of Abnormal CD8+ T cell proportion HP:0031393.
Abnormal naive CD8+ T cell proportion HP:0410374
Abnormal memory CD8+ T cell proportion HP:0410380
Abnormal HLA DR+ CD8+ T cell proportion HP:0025842

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-1526-4557

**Comment by pnrobinson** [comment]
Sorry, forgot this last time. I implemented as suggested.
